react/src/components/lablupTalkativotUI/LLMChatCard.tsx

```diff
@@ -240,9 +240,6 @@ const LLMChatCard: React.FC<LLMChatCardProps> = ({
           size="small"
           requiredMark="optional"
           style={{ flex: 1 }}
-          initialValues={{
-            baseURL: baseURL,
-          }}
```

Comment:
These lines of code prevent the baseURL from being automatically inserted by default. Is this the intended?

Reply:
I have submitted the handling of `initialValues`. Now, the `Form` has a key prop to ensure that the baseURL changes when the user changes the selected endpoint. Please check.